databrowser/src/components/table/TableCustom.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div className="overflow-x-auto">
+    <div class="overflow-hidden min-w-full rounded-xl border border-gray-400">
```

Comment:
Due to overflow-hidden, the mobile view does not scroll and on the x-axis. We need a another way to handle this.

Reply:
You could try to fix it with pseudo classes `first, last` element.

https://tailwindcss.com/docs/hover-focus-and-other-states#first-child